packages/builders/scripts/vats/generic.js

```diff
@@ -0,0 +1,27 @@
+const strcmp = (a, b) => (a < b ? -1 : a > b ? 1 : 0);
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').CoreEvalBuilder} */
+export const genericProposalBuilder = async (
```

Comment:
```suggestion
export const vatUpgradeProposalBuilder = async (
```
(along with renaming the file to e.g. packages/builders/scripts/vats/upgrade.js)

Reply:
I like making the name more specific, but I chose `packages/builders/scripts/vats/upgrade-vats.js` and:
```js
export const upgradeVatsProposalBuilder ...
```

for consistency.